### PR TITLE
feat: Restart function on update modal

### PR DESCRIPTION
**Pull request by Segergren**
- Restart functionality
- Click 'Confirm' to update
- Vertically center modal icon
- Fixed no thumbnails on SelectFolder change

Download SVG
download

### CODE REVIEW

File: Classes/Utils/Messages.cs [hunk 1]
```diff
@@ -427,8 +427,23 @@ public class WebMessage {
                         };
 
                         Process.Start(processInfo);
+                        Process.GetCurrentProcess().Kill(); // this is not a clean exit, need to look into why we can't cleanly exit
+
+#else
+                        string applicationPath = Environment.GetCommandLineArgs()[0];
+                        string escapedApplicationPath = applicationPath.Replace("\"", "\\\"");
```
Comment (lulzsun) [inline Classes/Utils/Messages.cs:433]: Linux support should be removed. Linux application updates may be handled differently in the future so there is no guarantee that we will be needing this.

Comment (Segergren) [reply Classes/Utils/Messages.cs:433]: Aight, I'll fix it tomorrow. I just turned off the PC 😀

File: Classes/Utils/Messages.cs [hunk 2]
```diff
@@ -427,8 +427,23 @@ public class WebMessage {
...
+                        string command = $"nohup \"{escapedApplicationPath}\" > /dev/null 2>&1 &";
```
[the remark this answers is not in the document]
Comment (Segergren) [reply Classes/Utils/Messages.cs]: I removed the linux support :)

**Review by lulzsun — APPROVED** [review]
Thanks for the changes! Ready to merge.

**Pull request merged**